e2e/fixtures.mjs

```diff
@@ -0,0 +1,55 @@
+/* eslint-env node */
+import dotenv from 'dotenv';
+import { StreamChat } from 'stream-chat';
+dotenv.config();
+dotenv.config({ path: `.env.local` });
+
+(async () => {
+  const {
+    ADD_MESSAGE_CHANNEL,
+    APP_KEY,
+    APP_SECRET,
+    JUMP_TO_MESSAGE_CHANNEL,
+    TEST_USER_1,
+    TEST_USER_2,
+  } = Object.entries(process.env).reduce((acc, [key, value]) => {
+    acc[key.replace('E2E_', '')] = value;
+    return acc;
+  }, {});
```

Comment:
Also not sure about this one but consider it a nitpick - if it's going to be used only once I think it's fine for the variables to have `E2E_` at the beginning. :)

Reply:
I'd agree. I've replaced this with a direct destructure now.